### DISCUSSION TITLE
Explorer (root directory) not working

### DISCUSSION THREAD

**Discussion by prabanjanraja** [LazyVim/LazyVim#1430] [Q&A]
I am using a windows 11 machine
whenever I try to use <leader>ee it just shows the name of the first directory next to the drive and nothing else but <leader>eE seems to work fine
why is that?

**Comment by dpetka2001** [comment]
`<leader>e` is supposed to open the `root_dir` according to the LSP attached on your buffer or resort to `cwd` if no LSP attached.
